lib/distributions/Hypersphere/BinghamDistribution.m

```diff
@@ -696,27 +703,79 @@
             end
         end
         
+        function B = fitScoreBased(samples, weights)

```

Comment:
Did you test if the score based fitting actually works? The last time I used it, there were still some bugs, I believe.

Reply:
This was copied over by accident, sorry. I have removed it from the commit.